public/stripejs-v3-mock.js

```diff
@@ -0,0 +1,50 @@
+// StripeJS fixture for using Stripe in feature specs. Mimics credit card form and Element objects.
+// Based on: https://github.com/thoughtbot/fake_stripe/blob/v0.3.0/lib/fake_stripe/assets/v3.js
```

Comment:
if this is your rewrite of https://github.com/thoughtbot/fake_stripe/blob/v0.3.0/lib/fake_stripe/assets/v3.js
I'd commit exactly what's there first and then a separate commit with your changes explaining why you did them.